src/tblib/__init__.py

```diff
@@ -61,24 +55,22 @@ def as_traceback(self):
             return tproxy(TracebackType, self.__tproxy_handler)
         elif tb_set_next:
             f_code = self.tb_frame.f_code
-            code = compile('\n' * (self.tb_lineno - 1) + 'raise __traceback_maker', self.tb_frame.f_code.co_filename, 'exec')
+            code = compile('\n' * (self.tb_lineno - 1) + 'raise Exception', self.tb_frame.f_code.co_filename, 'exec')
             if PY3:
                 code = CodeType(
-                    0, 0,
-                    f_code.co_nlocals, f_code.co_stacksize, f_code.co_flags,
+                    0, code.co_kwonlyargcount,
```

Comment:
Probably not as it should always be 0 anyway, but I'm just making this entire block match up with the implementation in jinja2's debug.py.